dependency_links.txt

```diff
@@ -0,0 +1 @@
+https://github.com/Rapptz/discord.py/tarball/8ccb98d395537b1c9acc187e1647dfdd07bb831b#egg=discord.py
```

Comment:
This needs to be listed as
```
https://github.com/Rapptz/discord.py/tarball/8ccb98d395537b1c9acc187e1647dfdd07bb831b#egg=discord.py-1.0.0a
```
or else this: https://gist.github.com/mikeshardmind/2c84190bc3d972e18b5566928a4627f6
happens

I'm also going to continue the prior recommendation on this being changed to the below, but the above is sufficient.
```
git+https://github.com/Rapptz/discord.py@8ccb98d395537b1c9acc187e1647dfdd07bb831b#egg=discord.py-1.0.0a
```

Reply:
Good catch @mikeshardmind, just fixed that up.